site/docs/contribute.md

```diff
@@ -45,6 +45,16 @@ The Iceberg community prefers to receive contributions as [Github pull requests]
 * If a PR is related to an issue, adding `Closes #1234` in the PR description will automatically close the issue and helps keep the project clean
 * If a PR is posted for visibility and isn't necessarily ready for review or merging, be sure to convert the PR to a draft
 
+### Merging Pull Requests
+
+Most pull requests can be merged once a committer is satisfied with the code in the PR. Before merging all review comments should be addressed either by making changes or agreeing the request is out of scope for the PR. For larger changes or additions to public APIs committers will wait at least 24 hours before merging to ensure there is no additional feedback from members of the community. As in all ASF governed projects committers are expected to act in the best interest of the project. If a committer feels there might be a conflict of interest with a pull request they review, they are encouraged to ask for another committer to look at the pull request.
```

Comment:
nit: this is a long line. various file editors will present it as is, with horizontal scroll
would it make sense to split the text into multiple lines?

Reply:
I tend to prefer multiple lines but I think the general style in this project is to use long lines for paragraphs instead of splitting them up in markdown files.  I was already asked to remove line breaks in this PR,

Reply:
I don't think we have any defined Markdown style yet, if you check what is in `spec.md` vs what is in `puffin-spec.md` they are completely different... Maybe we should just add a markdown linter after this...

Reply:
A linter would be very much appreciated, I'd prefer to avoid this particular style debate on this PR though.

Reply:
+1

Reply:
Added #10790